optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -273,6 +273,8 @@ def render(self, result, job):
             os.makedirs(html_dir)
             html_path = os.path.join(html_dir, 'results.html')
             self._render(result, html_path)
+            html_symlink_path = os.path.join(job.logdir, 'results.html')
+            os.symlink(os.path.join('html', 'results.html'), html_symlink_path)
```

Comment:
Yeah, I thought about that too... but was a bit weare about touching the *real* file location at this point.  I can definitely change it... let me see what others feel about it.